packages/atoms/src/InputWrapper.tsx

```diff
@@ -0,0 +1,56 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import React, { ReactNode } from 'react';
+import { View } from 'react-native';
+import { Box } from './Box';
+import { Icon } from './Icon/Icon';
+import { IconType } from './Icon/iconTypes';
+
+type InputWrapperProps = {
```

Comment:
honestly, I am still getting confused by props being at the top :C And the fact that it's different on desktop and mobile is not very cool imo

Reply:
I have to say, that for me it's also better to place prop types to the top of component because sometimes I just want to check file to see what props I need to pass to component and I don't want to look for them under the styles, which shouldn't bother me at that moment. But I see that on suite desktop it is being placed right above the component. 

Maybe I'll put it right above the component and then we can decide on code-improvements whether we should change it or not? It will be better to have it consistent throughout the whole project for people being able to move between packages without having to change to other code styles / component structures.

Reply:
Fixed

Reply:
Now we have one file that is different from all others in package. Put it back to the top pls, at least keep consistency per package which is fine.

Reply:
[5895f89](https://github.com/trezor/trezor-suite/pull/5433/commits/5895f89b726e8cc07e9f5fd0be96d7fc97bb8c71)